policy/centos7/rancher.te

```diff
@@ -1,4 +1,4 @@
-policy_module(rancher, 1.0.0)
+policy_module(rancher, 1.0.1)
```

Comment:
I don't think this needs to be bumped, at least it never changes in rke2-selinux. @dweomer thoughts?

Reply:
This version is disconnected from the artifacts that we release, which isn't great, but until we have a process to ensure such can be in sync we can avoid bumping it here (in-place upgrades seem to work just fine with same version).